sdk/python/feast/feature_store.py

```diff
@@ -149,6 +122,7 @@ def __init__(
 
         registry_config = self.config.get_registry_config()
         self._registry = Registry(registry_config, repo_path=self.repo_path)
+        self._registry._initialize_registry()
```

Comment:
not sure why originally separated - moved this to Registry constructor

Reply:
actually moved back to `repo_operations.py`, since we don't want to always initialize the `Registry` - there's a test in `test_online_retrieval.py` intended to ensure that if `registry.db` is deleted, an error is thrown:
```
# Force registry reload (should fail because file is missing)
        with pytest.raises(FileNotFoundError):
            fs_infinite_ttl.refresh_registry()
```